catalog/rest/catalog-rest-service/pom.xml

```diff
@@ -38,6 +39,12 @@
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-frontend-jaxrs</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.catalog.core</groupId>
+            <artifactId>catalog-core-attachment</artifactId>
+            <version>2.14.0-SNAPSHOT</version>
+            <scope>compile</scope>
```

Comment:
Hard coded version and don't need the default scope.

Reply:
Yes, need to make sure the property is referencing 2.13.3-SNAPSHOT

Reply:
Removed scope and changed to `${project.version}`.